api/usePlan.test.js

```diff
@@ -132,7 +132,7 @@ describe('usePlan', () => {
       expect.stringContaining(`/plans/${expectedPlan.id}/actions/${actionId}`),
       expect.objectContaining({
         method: 'PATCH',
-        body: JSON.stringify({ isCompleted: true }),
+        body: JSON.stringify({ isCompleted: true, completedBy: '' }),
```

Comment:
Any reason we can't add "completedBy" to the request on line 126 and check that it gets set correctly here - rather than checking for an empty string?

Reply:
No reason, makes sense !